app/app.go

```diff
@@ -1,8 +1,13 @@
 package app
 
 import (
+	"context"
+	"cosmossdk.io/x/tx/signing"
 	"encoding/json"
 	"fmt"
+	"github.com/cosmos/cosmos-sdk/codec/address"
+	"github.com/cosmos/cosmos-sdk/std"
+	"github.com/cosmos/gogoproto/proto"
```

Comment:
might be worth running our formatters for consistent import arrangement:
`make format`